packages/components/pie-button/src/index.ts

```diff
@@ -34,6 +34,18 @@ export class PieButton extends LitElement implements ButtonProps {
         this._internals = this.attachInternals();
     }
 
+    connectedCallback () {
+        super.connectedCallback();
+
+        this.form?.addEventListener('keydown', this._handleFormKeyDown);
+    }
+
+    disconnectedCallback () {
+        super.disconnectedCallback();
+
+        this.form?.removeEventListener('keydown', this._handleFormKeyDown);
+    }
+
```

Comment:
do you think we can pass `@keydown` to the button directly instead of this way? such as 

`<button @keydown={this. _handleFormKeyDown}`

Reply:
I believe we want to capture `keydown` on anything inside the form. I.e. pressing enter when on a text input to submit the form. Would attaching the listener to the button enable this still?

Reply:
sorry i didn't see this.form :,(